### DISCUSSION TITLE
Feature request: show which tile(s) must be moved in "raise or lower land first" situations

### DISCUSSION THREAD

**Discussion by galfisk** [OpenRCT2/OpenRCT2#16648] [Feature requests]
When trying to place a multi-tile ride element which intersects ground or water, it would be useful if the game highlighted which of the tile(s) are causing the issue. Maybe flashing the tile border in error-message red, spade-tool white or water-tool teal. Or if not flashing, highlighting it as long as the error message is visible.

**Comment by duncanspumpkin** [comment]
This would be hard to achieve due to how errors are handled.
